src/potions.ts

```diff
@@ -211,7 +211,6 @@ export class Potion {
   }
 
   meatDrop(): number {
-    setLocation($location`Friar Ceremony Location`);
     return (
       getModifier("Meat Drop", this.effect()) +
       2 * (usingPurse() ? getModifier("Smithsness", this.effect()) : 0)
```

Comment:
I feel like `meatDrop()` should be independent of what our previous adventured location was. We could always do a
```
const prevLocation = myLocation();
setLocation($location.none);
const computedMeatDrop = getModifier("Meat Drop", this.effect()) +
      2 * (usingPurse() ? getModifier("Smithsness", this.effect()) : 0);
setLocation(myLocation);
return computedMeatDrop;
```

We handle barf-specific buffs in mood, while potions is also used in yachtzees (including for nobarf). I also wouldn't want to start grabbing underwater meat% immediately after yachtzees.